src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/Readcomic.kt

```diff
@@ -0,0 +1,201 @@
+package eu.kanade.tachiyomi.extension.en.readcomic
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Readcomic : ParsedHttpSource() {
+
+    override val name = "ReadComic"
+
+    override val baseUrl = "https://readcomic.net"
+
+    override val lang = "en"
+
+    override val supportsLatest = true
+
+    override fun popularMangaSelector() = ".manga-box"
+
+    override fun latestUpdatesSelector() = ".home-list .hl-box .hlb-name"
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/popular-comic/$page")
```

Comment:
```suggestion
        return GET("$baseUrl/popular-comic/$page", headers)
```

here and everywhere else